nix/nixos/cardano-node-service.nix

```diff
@@ -513,6 +516,16 @@ in {
         '';
       };
 
+      useBootstrapPeers = mkOption {
+        type = types.nullOr (types.either types.bool (types.listOf types.attrs));
+        default = false;
+        description = ''
+          If set, it will enable bootstrap peers.
+          To disable set this to false.
```

Comment:
:thinking: so we parse `false`, maybe it's enough to leave it unset - then no extra parsing is required.

Reply:
I like having it so people looking at the source code can be clear what's it set to in default

Reply:
What if we interpret an empty list as `DontUseBootstrapPeers`?

Reply:
We'd loose the ability of configuring a node to only use local roots as bootstrap peers.

Reply:
This conversation is resolved, but note that one can repeat a local roots in bootstrap peers.

Reply:
I don't think that's avoidable. What we decided was that if that's the case that's a misconfiguration and not really our fault, right?